### PR TITLE
ISSUE #3225 - Viewer facelift - BIM Card

### LINKED ISSUE #3225: Viewer facelift - Bim Card
### Description
We need to revamp the BIM card to match v5 styles

### Goals
- [ ] To have the BIM card styled like v5 

### Tasks
- [ ] Just follow the styles in the mockup

### Related Resources
[MockUp](https://www.figma.com/file/J5I3hYQTrLCjEVP1iqcyZK/3D-Viewer?node-id=136292%3A177725)

### PR DESCRIPTION
**Pull request by The-Daniel**
This fixes #3225

#### Description
- Adds styling to the BIM card in the v5 viewer

#### Test cases
- Open the BIM viewer.
- Click different segments
- Favourite/Unfavourite items

### CODE REVIEW

**Comment by george-hadfield** [comment]
Waiting to get access to a model with BIM data

**Comment by sanmont3drepo** [comment]
Make the dotted border span the full width of the card-allowing for the side margins.

image

**Comment by sanmont3drepo** [comment]
The grey should be #F7F8FA
image

File: frontend/src/v5/ui/v4Adapter/overrides/bim.overrides.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
```
Comment (Amantini1997) [inline frontend/src/v5/ui/v4Adapter/overrides/bim.overrides.ts:2]: copyright year

**Comment by Amantini1997** [comment]
> Make the dotted border span the full width of the card-allowing for the side margins.
> 
> image

I reckon this is fixed in the base branch, you just need to do a pull

**Review by Amantini1997 — CHANGES_REQUESTED** [review]

**Review by george-hadfield — APPROVED** [review]

**Pull request merged**